addons/popochiu/engine/objects/character/popochiu_character.gd

```diff
@@ -757,59 +782,19 @@ func face_direction(destination: Vector2):
 	# Notice how we seek for opposite directions for left and
 	# right. Flipping is done in other functions. We just define
 	# a preference order for animations when available.
+#
+	# Get the vector from the origin to the destination.
+	var angle = wrapf(rad_to_deg((destination - position).angle()) , 0, 360)
+	# The angle calculation uses 16 angles rather than 8 for greater accuracy in choosing the facing
+	# direction animation preference.
+	var _looking_angle := int(angle / 22.5) % 16
+	_animation_suffixes = _valid_animation_suffixes[_looking_angle] + ['']
+	# The 16 directions used for animation suffixes are simplified to 8 general directions
+	_looking_dir = int(angle / 45) % 8
+	if _looking_dir == _last_looking_dir:
+		return
```

Comment:
This is redundant to me. Of course, you avoid an assignment, but it's idempotent, so I would keep it simple.

Reply:
Removed these two lines as requested.